api/server.spec.js

```diff
@@ -0,0 +1,156 @@
+const request = require('supertest');
+const server = require('./server');
+const db = require('../data/dbConfig');
+
+describe('the route handlers', () => {
+
+    beforeEach( () => {
+        return db.migrate.rollback()
+            .then( () => {
+                return db.migrate.latest()
+                    .then( () => {
+                        return db.seed.run()
+                    })
+            })
+    })
+
+    afterEach(async () => {
+        await db.migrate.rollback()
+    });
+
+    describe('get /', () => {
+
+        it('responds with status 200', async () => {
+            const success = await request(server).get('/')
+
+            expect(success.status).toBe(200)
+        })
+
+        it('sends the correct response', async () => {
+            const success = await request(server).get('/')
+
+            expect(success.body.length).toBe(5)
+        })
+
+
+        it('sends an empty array if nothing in the database', async () => {
+            await request(server).delete('/1')
+            await request(server).delete('/2')
+            await request(server).delete('/3')
+            await request(server).delete('/4')
+            await request(server).delete('/5')
+            const success = await request(server).get('/')
+
+            expect(success.body).toEqual([])
+        })
+    })
```

Comment:
If you were to incorporate the object hardcoded into the testing file you could also add these tests. The object would have to maintain the structure of the code in the database. 
`it("should return a list of games", async () => {
      const response = await request(server).get("/games");
      expect(response.body).toEqual(games);
    });`


   ` it("should return an array", async () => {
      const response = await request(server).get("/games");
      expect(Array.isArray(games)).toBeTruthy();
    });`